osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -1330,10 +1358,8 @@ namespace renderkit {
                     glDisable(GL_STENCIL_TEST);
                 }
             }
-            if (checkForGLError(
```

Comment:
Why is this check/return removed?  Is it okay for an OpenGL error to happen here?

Reply:
oh! yeah, sorry, I'm not fully used to staging changes from VS Code, accidentally included this one.  This was a warning that "control reached end of non-void lambda without returning a value" - no other code paths return a value (and the "finally" util doesn't actually use a return value from the functor in any way) but because one code path returned a value (this conditional) it made the whole thing a bool functor.

If you actually want the surrounding function to return false in the case of an error, then you'll need to capture an outside success bool by reference and set that, not just return from the lambda.

Reply:
Yummy yummy Lambda functions.  Probably a copy-paste error.